canonical-web-and-design/jaas-dashboard
# Add application details page

## Done

- Add application details page
- Refactor structure of ModelDetails to EntityDetails
- Rename Controllers and Models to ControllersIndex and ModelsIndex respectively
- 

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Go to a model, click through to details, click an app.. view a page, not a panel.
- Check everything

## Details

Fixes https://github.com/canonical-web-and-design/juju-squad/issues/1622
